extra/1-factorial.js

```diff
@@ -10,6 +10,11 @@
 
 function factorial(input) {
   // TODO
+  var factored = 1;
```

Comment:
Hey Nasir, great job on getting all the exercises done! Just a little suggestion - maybe try to replace` var` with `const` and `let ` in the future? I think the new convention (since ES6) is to use them instead of` var `.  https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/